code/modules/jobs/jobs.dm

```diff
@@ -1,3 +1,38 @@
+GLOBAL_LIST_INIT(station_positions, list(

```

Comment:
This list HAS to already exist somewhere

Reply:
I'll take a second look

Reply:
i swear, this doesn't exist somehow

Reply:
This should work https://github.com/tgstation/tgstation/blob/a89c5aee03f3660d31715f268b192270e90abeae/code/controllers/subsystem/job.dm#L7

Reply:
i'll test it tonight to make sure, thanks for the help

Reply:
wow, this is great. I should really refactor a lot of the role checking code to work with the job subsystem